tests/Makefile

```diff
@@ -94,14 +88,22 @@ ifeq ($(MYST_ENABLE_HOSTFS),1)
 DIRS += hostfs
 endif
 
-DIRS += msync
+ifndef MYST_ENABLE_GCOV
```

Comment:
This is a really tricky conditional expression but I can't think of anything better.

Reply:
Agreed